### PR TITLE
Panel Discussion 1st Draft

### PR DESCRIPTION
**Pull request by jcjudkins**
Need a working title.

### CODE REVIEW

File: src/_content/posts/discussion-panel.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+author: DjangoCon US Communication Team
+category: Program
+date: 2024-09-06 06:00:00-04:00
+image: null
+layout: post
+title: 'Announcing Our Panel Discussion: Who put me in charge? Moving beyond day-to-day
+  coding in Django'
+---
+
+# Announcing our panel discussion for DjangoCon US 2023: Who put me in charge? Moving beyond day-to-day coding in Django
```
Comment (tim-schilling) [inline src/_content/posts/discussion-panel.md]:
```suggestion
# Announcing our panel discussion for DjangoCon US 2024: Open, friendly, and welcoming: on the history and future of mentoring in Django
```

File: src/_content/posts/discussion-panel.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+author: DjangoCon US Communication Team
+category: Program
+date: 2024-09-06 06:00:00-04:00
+image: null
+layout: post
+title: 'Announcing Our Panel Discussion: Who put me in charge? Moving beyond day-to-day
```
Comment (tim-schilling) [inline src/_content/posts/discussion-panel.md]:
```suggestion
title: 'Announcing Our Panel Discussion: Open, friendly, and welcoming: on the history and future of mentoring in Django
```

File: src/_content/posts/discussion-panel.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  coding in Django'
+---
+
+# Announcing our panel discussion for DjangoCon US 2023: Who put me in charge? Moving beyond day-to-day coding in Django
+
+We're excited to announce our panel discussion for DjangoCon US 2024!
+
+Lacey Henschel, will host a panel discussion  about the ROI for the smaller communities on the larger django community and we need to keep doing it, and help make them better.
```
Comment (tim-schilling) [inline src/_content/posts/discussion-panel.md]:
```suggestion
Lacey Henschel, will host a panel discussion about the impact smaller communities have on the larger Django community, why we need to keep doing it, and help make them better.
```

File: src/_content/posts/discussion-panel.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+author: DjangoCon US Communication Team
+category: Program
+date: 2024-09-06 06:00:00-04:00
+image: null
+layout: post
+title: 'Announcing Our Panel Discussion: Who put me in charge? Moving beyond day-to-day
+  coding in Django'
```
Comment (drewbrew) [inline src/_content/posts/discussion-panel.md]:
```suggestion
title: 'Announcing Our Panel Discussion: Open, friendly, and welcoming: On the history and future of mentoring in Django'
```

File: src/_content/posts/discussion-panel.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+author: DjangoCon US Communication Team
+category: Program
+date: 2024-09-06 06:00:00-04:00
+image: null
+layout: post
+title: 'Panel Discussion: Open, friendly, and welcoming: on the history and future of mentoring in Django'
+---
+
+# Announcing our panel discussion for DjangoCon US 2023: Panel Discussion: Open, friendly, and welcoming: on the history and future of mentoring in Django
```
Comment (tim-schilling) [inline src/_content/posts/discussion-panel.md]:
```suggestion
# Announcing our panel discussion for DjangoCon US 2024: Panel Discussion: Open, friendly, and welcoming: on the history and future of mentoring in Django
```

File: src/_content/posts/discussion-panel.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+We're excited to announce our panel discussion for DjangoCon US 2024!
+
+acey Henschel will moderate a panel discussion on the history and future of communities within the Django community.
```
Comment (drewbrew) [inline src/_content/posts/discussion-panel.md]:
```suggestion
Lacey Henschel will moderate a panel discussion on the history and future of communities within the Django community.
```

Comment (tim-schilling) [inline src/_content/posts/discussion-panel.md]:
```suggestion
Lacey Henschel will moderate a panel discussion on the history and future of communities within the Django community.
```

**Review by drewbrew — APPROVED** [review]

File: src/_content/posts/discussion-panel.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+We'll explore how mentoring, community involvement, and support networks are important in keeping Django resilient. By highlighting the work of groups like Django Girls, PyLadies, Djangonaut Space, and Black Python Devs, we’ll see how these initiatives not only foster personal growth but also play a vital role in keeping the Django community strong and sustainable for the long term.
```
Comment (tim-schilling) [inline src/_content/posts/discussion-panel.md]:
```suggestion
We'll explore how mentoring, community involvement, and support networks are important in keeping Django resilient. By highlighting the work of groups like [Django Girls](https://djangogirls.org), [PyLadies](https://pyladies.com), [Djangonaut Space](https://djangonaut.space), and [Black Python Devs](https://blackpythondevs.com/), we’ll see how these initiatives not only foster personal growth but also play a vital role in keeping the Django community strong and sustainable for the long term.
```

**Comment by tim-schilling** [comment]
Hmm, I was trying to rebase on the other. Please hold

**Comment by tim-schilling** [comment]
There we go. Sorry for the hassle

**Pull request merged**